ghpages_deploy.sh

```diff
@@ -42,7 +42,9 @@ if ! git checkout gh-pages ; then
 fi
 
 # Overwrite files in the branch
-git rm -rfq ./*
```

Comment:
No, it doesn't. I tried many things with git rm. Nothing worked. Hence had to shift to this